onnxruntime/test/providers/cpu/nn/conv_op_test.cc

```diff
@@ -24,7 +24,7 @@ void TestConvOp(const ConvOpAndTestAttributes& attributes,
                 const vector<vector<int64_t>>& input_shapes,
                 const std::initializer_list<float>& expected_output,
                 const vector<int64_t>& expected_output_shape,
-                bool weight_is_initializer = false,
+                bool weight_is_initializer = true,
```

Comment:
It's better to update the individual test which is applicable here, also we want to test both weight_is_initializer is true or false

Reply:
Updated.